### PR TITLE
[taheryasir/Step 11]: Let's set the validation

### PR DESCRIPTION
**Pull request by taheryasir**
Task Description:
https://github.com/Fablic/training/blob/develop_en/README.md#step-11-lets-set-the-validation

removed field require params from `_form.html.erb` to check model validations

to run model spec use
`docker exec -it myapp_api_1 bundle exec rspec spec/models`

output of validations
Screen Shot 2021-06-18 at 14 15 42

### CODE REVIEW

**Review by yo-it-engineer — COMMENTED** [review]
There is only one minor mistake. The rest seemed good!

File: myapp/spec/models/task_model_spec.rb [hunk 1]
```diff
@@ -9,77 +9,49 @@
     let(:params) { { name: name, desc: desc, status: status, label: label, priority: priority } }
```
Comment (yo-it-engineer) [inline myapp/spec/models/task_model_spec.rb:9]: 【just some explanation】What I meant is to set the correct params here.
Then just update part of the parameter inside each "context" like in L17, L24...

But this current way also work!

**Review by yo-it-engineer — APPROVED** [review]
I see you do the rubocop too. Thank for that.

LGTM!
Image from iOS (88)

File: myapp/spec/models/task_model_spec.rb (hunk 1 above)
Comment (taheryasir) [reply myapp/spec/models/task_model_spec.rb:9]: ohh OK, will do that in next PR then.

**Pull request merged**